### ISSUE TITLE
Events / Hooks

### ISSUE THREAD

**Issue by rkumorek** [nextapps-de/spotlight#2]
Hey,

Are you planning to add events / hooks so for example we could run callback after slide has changed?

I'd like to integrate Spotlight with carousel but I can't see a way to synchronize carousel current element after interacting with lightbox. With like `changed` or `updated` event it would be easily done.

**Comment by ts-thomas** [comment]
Hello. This is now available (v0.6.11). From the readme:

Option: "onchange", Type: function(index)
> Pass a callback function which is get fired every time when a page has changed (the first parameter is equal to the new index). __Note:__ The image may not have been fully loaded when the event is fired (preloading phase).

Use Spotlight by API, e.g.:

```js
var gallery = [{
    title: "Image 1",
    description: "This is a description.",
    src: "gallery/london-1758181.jpg"
},{
    title: "Image 2",
    description: "This is a description.",
    src: "gallery/sea-1975403.jpg"
},{
    title: "Image 3",
    description: "This is a description.",
    src: "gallery/newport-beach-2089906.jpg"
}];
```

```js
Spotlight.show(gallery, {
    index: 1,
    onchange: function(index){
        console.log("Current index: " + index);
    }
});
```

**Comment by ts-thomas** [comment]
Sorry, there was an issue with the __onchange__ event, it is now fixed in v0.6.12
